feathr_project/feathr/_feature_registry.py

```diff
@@ -807,3 +817,60 @@ def get_features_from_registry(self, project_name: str, workspace_path: str):
         with open(feature_gen_file, "w") as online_config:
             online_config.write(feature_gen_conf_content)
         logger.info("Writing online configuration from feathr registry to {}", feature_gen_file)
+   
+    def get_feature_by_fqdn_type(self, qualifiedName, typeName):
+        """
+        Get a single feature by it's QualifiedName and Type
+        Returns the feature else throws an AtlasException with 400 error code
+        """
+        response = self.purview_client.get_entity(qualifiedName=qualifiedName, typeName=typeName)
+        entities = response.get('entities')
+        for entity in entities:
+            if entity.get('typeName') == typeName and entity.get('attributes').get('qualifiedName') == qualifiedName: 
+                return entity
+       
+    def get_feature_by_fqdn(self, qualifiedName):
+        """
+        Get feature by qualifiedName
+        Returns the feature else throws an AtlasException with 400 error code
+        """        
+        guid = self.get_feature_guid(qualifiedName)
+        return self.get_feature_by_guid(guid)
```

Comment:
could you point out where FQDN is defined?

Reply:
https://github.com/linkedin/feathr/blob/c4266bb7d70865a7846a6ae785c9a831ac554de4/feathr_project/feathr/_feature_registry.py#L190 It is defined as part of registering features.